### ISSUE TITLE
Would it be possible to allow the plugin to use a main file linked to included files?

### ISSUE THREAD

**Issue by thatguynm** [Graham-Hayes/AC-Ink-Integration#5]
I know this isn't much of an issue, but I wasn't sure where to post this.

I'm working on a project that could be an organizational headache if I tried using a single Ink file to include all the text.

Is it feasible to have the program read from included files if the main file was attached in the persistent engine, and all necessary story files were in the same folder as the main Ink file? Or is this more a limitation of the Ink integration into Unity?

**Comment by Graham-Hayes** [comment]
If that doesn't work out of the box I believe that is on the Ink end. As I understand it, the includes are for humans but ink should make it as if it was one file.


-------- Original message --------

**Comment by thatguynm** [comment]
You are right, I just managed to get it working. I had my variable declarations above my includes, which was breaking everything. Your response helped me isolate the problem and fix it. Thank you
